suse_openstack_cloud

```diff
@@ -390,3 +391,18 @@ if [ -e /opt/dell/crowbar_framework/.crowbar-installed-ok ]; then
   find_and_plog_files_0 $chef_export_dir -type f
   rm -rf $chef_export_dir
 fi
+
+###############################################################
+section_header "RabbitMQ broker"
+
+if [ -e /usr/sbin/rabbitmqctl ]; then
+    rabbitmq_tmpdir=$(mktemp -d)
+    rabbitmq_report_log=$rabbitmq_tmpdir/rabbitmq_report.log
+    source /etc/rabbitmq/rabbitmq-env.conf
+    pushd `pwd`
```

Comment:
Code was changes in https://github.com/SUSE-Cloud/supportutils-plugin-suse-openstack-cloud/pull/47